test/functional/interface_bitcoin_cli.py

```diff
@@ -59,7 +59,6 @@ def run_test(self):
         assert_equal(cli_get_info['protocolversion'], network_info['protocolversion'])
         if self.is_wallet_compiled():
             assert_equal(cli_get_info['walletversion'], wallet_info['walletversion'])
-            assert_equal(cli_get_info['balance'], wallet_info['balance'])
             assert_equal(cli_get_info['coinjoin_balance'], wallet_info['coinjoin_balance'])
```

Comment:
15564 this needs to be moved down

Reply:
should be present on line 70

Reply:
walletversion and coinjoin_balance need to be moved down